### ISSUE TITLE
Consolidate udev rules

### ISSUE THREAD

**Issue by samiebee43** [WasatchPhotonics/Wasatch.PY#85]
The Enlighten version of this file seems to have more stuff in it, but the labels are inconsistent all around.

**Comment by mzieg** [comment]
Good call.  Initial step in https://github.com/WasatchPhotonics/ENLIGHTEN/pull/258
